jit/v2/JIT_v2.0.0.yaml

```
@@ -547,22 +736,82 @@ paths:
                   summary: |
                     carrierServiceCode is missing
                   description: |
-                    `carrierServiceCode` is a mandatory property in the `Terminal Call`. This is an example of how the error object would look in case this property is missing
+                    `carrierServiceCode` is a mandatory property in the **Terminal Call**. This is an example of how the error object would look in case this property is missing
 
                     **NB**: `errorCode` not yet standardized by DCSA. Value `7003` is just a "random example"
                   value:
-                    httpMethod: PUT
-                    requestUri: /v2/terminal-calls/085a3207-5e45-49cf-8e1b-f8442beaf545
+                    httpMethod: 'PUT'
+                    requestUri: '/v2/terminal-calls/085a3207-5e45-49cf-8e1b-f8442beaf545'
                     statusCode: 400
-                    statusCodeText: Bad Request
-                    statusCodeMessage: carrierServiceCode not found - it is a mandatory property in Terminal Call
-                    providerCorrelationReference: 4426d965-0dd8-4005-8c63-dc68b01c4962
+                    statusCodeText: 'Bad Request'
+                    statusCodeMessage: 'carrierServiceCode not found - it is a mandatory property in Terminal Call'
+                    providerCorrelationReference: '4426d965-0dd8-4005-8c63-dc68b01c4962'
+                    errorDateTime: '2024-11-21T09:41:00Z'
+                    errors:
+                      - errorCode: 7003
+                        property: 'carrierServiceCode'
+                        errorCodeText: 'mandatory property missing'
+                        errorCodeMessage: 'carrierServiceCode must be provided as part of a Terminal Call'
+        '404':
+          description: |
+            In case the implementor does not know of the `portCallID` used in the request payload - it is possible for the implementor to reject the requests by returning a `404` (Not Found).
```

```suggestion
            In case the implementator does not know of the `portCallID` used in the request payload - it is possible for the implementator to reject the request by returning a `404` (Not Found).
```
Probably typos in other locations in the schema as well.

---

Fixed + all other places as well